lua/entities/pewpew_base_cannon/init.lua

```diff
@@ -205,11 +209,15 @@ end
 function ENT:NewSystem_FireBullet()
 	local Dir, Pos = pewpew:GetFireDirection( self.Direction, self, ent )
 	local num = self.Bullet.Spread or 0
-	if (num and num != 0) then
-		local randomang = Angle( math.Rand(-num,num), math.Rand(-num,num), math.Rand(-num,num) )
-		Dir:Rotate(randomang)
-	end
-	pewpew:FireBullet( Pos, Dir, self.Owner, self.Bullet, self, self.Direction )
+    if self.ShotCount then
```

Comment:
i would just use tabs if the rest of the project is tabs

Reply:
?